x-pack/plugins/monitoring/public/components/apm/instances/instances.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { get } from 'lodash';
+import { MonitoringTable } from '../../table';
+import {
+  KuiTableRowCell,
+  KuiTableRow
+} from '@kbn/ui-framework/components';
+import { SORT_ASCENDING } from '../../../../common/constants';
+import { EuiLink } from '@elastic/eui';
+import { Status } from './status';
+
+
+const filterFields = [ 'beat.name', 'beat.version' ];
+const columns = [
+  { title: 'Name', sortKey: 'beat.name', sortOrder: SORT_ASCENDING },
```

Comment:
Will we need this? In the case of APM there is only apm-server?

Reply:
True, we will mostly not need to represent this

Reply:
> apm-server was providing an initial document when it was "ready". Do we have still such a document

Yes that's still done for first setup but we can't rely on those being there as they are only created when apm-server starts up and may be pruned when cleaning up indexes.

> How many apm-server instances do we expect to be monitored?

Likely very few, even low double digits would be surprising.